Games/types/Mafia/roles/cards/WinIfAliveWhenTargetWin.js

```diff
@@ -0,0 +1,34 @@
+const Card = require("../../Card");
+const { PRIORITY_WIN_CHECK_DEFAULT } = require("../../const/Priority");
+const { PRIORITY_WIN_CHECK_DEFAULT } = require("../../const/Priority");
+
+module.exports = class WinIfAliveWhenTargetWin extends Card {
```

Comment:
is it possible to just set the role's .alignment? 

anyway I will hold out on commenting on this until you have tested the roles.